### PR TITLE
Added an http method parameter for request, populated by the urlRequest

### PR DESCRIPTION
**Pull request by br-alex-reyes**

### CODE REVIEW

**Review by br-tyler-milner — CHANGES_REQUESTED** [review]
@alexreyesBR this looks good. Can you go ahead and add an entry in the `CHANGELOG.md` too?

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 #### Enhancements
 
+* Added `method` parameter to `HTTP.Request`.ß
```
Comment (earlgaspard) [inline CHANGELOG.md:5]: Is the extra character at the end needed?

File: Sources/Hyperspace/HTTP/HTTP.swift [hunk 2]
```diff
@@ -162,6 +162,9 @@ public struct HTTP {
 
         /// The `URL` to which the `Request` originated.
         public let url: URL?
+        
+        /// The HTTP method for a request
```
Comment (earlgaspard) [inline Sources/Hyperspace/HTTP/HTTP.swift:166]: Please add a period at the end for consistency with other comments.
```suggestion
        /// The HTTP method for a request.
```

File: Sources/Hyperspace/HTTP/HTTP.swift [hunk 3]
```diff
@@ -172,18 +175,20 @@ public struct HTTP {
         /// Initialize a new `Request` with any given `URL`, HTTP headers and body.
         /// - Parameters:
         ///   - url: The `URL` for which this request was created.
+        ///   - method: The HTTP method for this request
```
Comment (earlgaspard) [inline Sources/Hyperspace/HTTP/HTTP.swift:178]: Minor - please add a period here too.
```suggestion
        ///   - method: The HTTP method for this request.
```

File: CHANGELOG.md (hunk 1 above)
Comment (br-tyler-milner) [inline CHANGELOG.md:5]: Looks like a minor typo here. We should remove the `ß` character.

**Review by br-tyler-milner — CHANGES_REQUESTED** [review]

**Review by earlgaspard — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (alexreyesBR) [reply CHANGELOG.md]: Nope, went ahead and removed it

**Review by earlgaspard — APPROVED** [review]

**Review by br-tyler-milner — DISMISSED** [review]
@earlgaspard I had to remove my personal GitHub account from the BR org on GitHub so you will need to merge this once the checks finish.

**Review by earlgaspard — APPROVED** [review]

**Pull request merged**